salt/modules/boto_ec2.py

```diff
@@ -55,6 +55,7 @@
 import salt.utils.compat
 import salt.ext.six as six
 from salt.exceptions import SaltInvocationError, CommandExecutionError
+from salt.utils import exactly_one
```

Comment:
Please change this to either `import salt.utils` and then use the fully qualified name when you call it or import with leading underscores to indicate private namspacing. I suggest the first option in this case.